app/_layouts/base.njk

```diff
@@ -14,7 +14,7 @@
 {% endblock %}
 
 {% block pageTitle %}
-  {% if title %}{{ title }} - {% endif %}{{ app.serviceName or app.productName }} - GOV.UK
+  {% if title %}{{ title }} - {% endif %}{{ app.serviceName or app.productName }}
```

Comment:
Instead of `GOV.UK` should we put `Department for Education`?
```suggestion
  {% if title %}{{ title }} - {% endif %}{{ app.serviceName or app.productName }} - Department for Education
```

Reply:
Good idea.